pkg/operator/ceph/csi/config.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2024 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package csi
+
+import (
+	"context"
+	"fmt"
+
+	csiopv1a1 "github.com/ceph/ceph-csi-operator/api/v1alpha1"
+	"github.com/pkg/errors"
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func CreateCephCSIOpConfCRRadosNS(ctx context.Context, c client.Client, cluster cephv1.CephCluster) error {
+
+	logger.Info("creating ceph-csi operator config cr for rados namespace")
+	cephRadosNamespaceList := &cephv1.CephBlockPoolRadosNamespaceList{}
+	err := c.List(ctx, cephRadosNamespaceList, &client.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	if len(cephRadosNamespaceList.Items) == 0 {
+		logger.Debug("no cephRadosNamespace is found")
+	} else {
```

Comment:
nit: Wherever you can return from the `if` block immediately, then no need to have the `else` block. It helps for code readability.
```suggestion
		return nil
	} 
```

Reply:
right, I was not returning after debug but I can return just after debug and remove else block